packages/mantine/src/components/table/Table.tsx

```diff
@@ -29,13 +29,20 @@ import {Th} from './Th';
 const useStyles = createStyles<string, {hasHeader: boolean}>((theme, {hasHeader}, getRef) => ({
     table: {
         width: '100%',
+        '& td:first-child': {
+            paddingLeft: 24,
```

Comment:
```suggestion
            paddingLeft: theme.spacing.md,
```

https://github.com/coveo/plasma/blob/master/packages/mantine/src/theme/Theme.tsx#L15